Runtime/Scripts/ObjImporter.cs

```diff
@@ -117,24 +120,11 @@ private void OnOBJRead(bool succes)
                 return;
             }
             createdGameobjectIsMoveable = !objreader.ObjectUsesRDCoordinates;
-
-
+            ObjImportSucceeded.Invoke(succes);
+            
             if (mtlFilePath != "")
```

Comment:
string.isnullorempty(value) misschien veiliger

Reply:
misschien, maar ik wil in deze PR geen wijzigingen doen aan andere zaken behalve de events